logging/context.go

```diff
@@ -0,0 +1,55 @@
+package logging
+
+import (
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+var (
+	internalLogLevel = map[LogLevel]zapcore.Level{ //nolint:gochecknoglobals
+		Debug: zapcore.DebugLevel,
+		Info:  zapcore.InfoLevel,
+		Warn:  zapcore.WarnLevel,
+		Error: zapcore.ErrorLevel,
+	}
+
+	locator = "stub" //nolint:gochecknoglobals
+
+	logLvl = zap.NewAtomicLevelAt(zap.DebugLevel) //nolint:gochecknoglobals
+
+	highPriority = zap.LevelEnablerFunc(func(lvl zapcore.Level) bool { //nolint:gochecknoglobals
+		return lvl >= zapcore.ErrorLevel && logLvl.Enabled(lvl)
+	})
+
+	lowPriority = zap.LevelEnablerFunc(func(lvl zapcore.Level) bool { //nolint:gochecknoglobals
+		return lvl < zapcore.ErrorLevel && logLvl.Enabled(lvl)
+	})
+
+	consoleEncoder zapcore.Encoder //nolint:gochecknoglobals
+	jsonEncoder    zapcore.Encoder //nolint:gochecknoglobals
+)
+
+func init() {
+	consoleEncoderConfig := zapcore.EncoderConfig{
+		TimeKey:        "Time",
+		LevelKey:       "Level",
+		NameKey:        zapcore.OmitKey,
+		CallerKey:      zapcore.OmitKey,
+		FunctionKey:    zapcore.OmitKey,
+		MessageKey:     "Message",
+		StacktraceKey:  zapcore.OmitKey,
+		LineEnding:     zapcore.DefaultLineEnding,
+		EncodeLevel:    zapcore.CapitalColorLevelEncoder,
+		EncodeTime:     zapcore.ISO8601TimeEncoder,
+		EncodeDuration: zapcore.StringDurationEncoder,
+		EncodeCaller:   zapcore.ShortCallerEncoder,
+	}
+	consoleEncoder = zapcore.NewConsoleEncoder(consoleEncoderConfig)
+	jsonEncoderConfig := consoleEncoderConfig
+	jsonEncoderConfig.EncodeLevel = zapcore.CapitalLevelEncoder
+	jsonEncoder = zapcore.NewJSONEncoder(jsonEncoderConfig)
+}
+
+func InitLocator(loc string) {
```

Comment:
@stepanvalyavskiy I'd change it to `Init()` and add a check if this function was called previously, and `panic()` if so.

Reply:
@stepanvalyavskiy also, we may need more params later, not only node locator hex string, so I'd pass the whole `*locators.Node` here